recipes/hdrhistogram-c/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class QuickfixConan(ConanFile):
+    name = "hdrhistogram-c"
+    license = "BSD 2-Clause License"
```

Comment:
```suggestion
    license = ("BSD-2-Clause", "CC0-1.0")
```

Short term version: https://spdx.org/licenses/BSD-2-Clause.html